template/express-rest-api/package.json

```
@@ -0,0 +1,32 @@
+{
+  "dependencies": {
+    "express": "^4.17.1",
+    "skuba-dive": "^1.1.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.7",
+    "@types/node": "^14.0.27",
+    "@types/supertest": "^2.0.10",
+    "skuba": "^3.7.7",
+    "supertest": "^4.0.2",
+    "typescript": "^3.9.7"
+  },
+  "engines": {
+    "node": ">=12"
+  },
+  "license": "UNLICENSED",
+  "private": true,
+  "scripts": {
+    "build": "skuba build",
+    "format": "skuba format",
+    "lint": "skuba lint",
+    "start": "skuba start",
+    "test": "skuba test"
+  },
+  "skuba": {
+    "entryPoint": "src/main.ts",
+    "template": "koa-rest-api",
+    "type": "application",
+    "version": "3.7.7"
+  }
```

This stuff is generated

```suggestion
  }
```

---

Sorry, if i get you correctly. Should I remove this line ? 😕 
 How about the entry point ?

---

The whole `skuba` section can be removed, you'll notice that none of the other templates have it. It's generated dynamically in part from the `skuba.template.js`, e.g.

https://github.com/mieshalannair/skuba/blob/express-template/template/express-rest-api/skuba.template.js#L6

I believe that specified entry point is mismatched with your template. Is there a reason to deviate and choose `main.ts` when every other template uses `app.ts`?

---

Noted with thanks. Will make the changes.
`main.ts` is usually used in the nestjs framework, but I think `app.ts` will do.